### PR TITLE
Deployment Guide

### PR DESCRIPTION
**Pull request by ross-p-smith**
Added a deployment guide around deploying these samples to mCCF

Relates to #66

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+!!! Note
+    The following commands assume you have created some pem files and they are in the deploy folder.
+
+```bash
+cd deploy
+
+# Create a json array of members in the correct format with the certificate
+certFile="member0_cert.pem"
+certAsJSONCompatibleString=$(< $certFile sed '$!G' | paste -sd '\\n' -)
+jsonArrayOfMembers="[{\"cert\":\"$certAsJSONCompatibleString\", \"encryptionKey\":\"\"}]"
```
Comment (julioalex-rezende) [inline deploy/README.md:36]: is `encriptionKey` the correct json tag (is it supposed to be empty?) ? or this has no relation to the template we use for adding a member?
( just fyi, in the template we have `encription_pub_key` )

Just drop this if not applicable

File: deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+az deployment group create \
+    --resource-group myResourceGroup \
+    --template-file bicep/mccf.bicep \
+    --parameters resourceName=myccf \
+        mccfMemberBasedSecurityPrincipals="$jsonArrayOfMembers"
+```
+
+## Deploying a CCF application to Azure Managed CCF
+
+Deploying a CCF application to Azure Managed CCF is very similar to deploying a CCF application to a local CCF instance. You can deploy any of the samples in this repository to Azure Managed CCF. The following commands will execute the same commands that are run in our Continuous Deplyment.
```
Comment (julioalex-rezende) [inline deploy/README.md:49]: nit:
Typo: Deployment

**Review by julioalex-rezende — APPROVED** [review]

File: deploy/README.md (hunk 1 above)
Comment (ross-p-smith) [reply deploy/README.md:36]: This is the attribute defined by the Azure Resource Manager and not us.

File: deploy/README.md (hunk 2 above)
Comment (ross-p-smith) [inline deploy/README.md:49]:
```suggestion
Deploying a CCF application to Azure Managed CCF is very similar to deploying a CCF application to a local CCF instance. You can deploy any of the samples in this repository to Azure Managed CCF. The following commands will execute the same commands that are run in our Continuous Deployment.
```

**Pull request merged**